src/V3EmitCMake.cpp

```diff
@@ -116,7 +116,7 @@ class CMakeEmitter final {
         cmake_set_raw(*of, name + "_COVERAGE", v3Global.opt.coverage() ? "1" : "0");
         *of << "# Timing mode?  0/1\n";
         cmake_set_raw(*of, name + "_TIMING", v3Global.usesTiming() ? "1" : "0");
-        *of << "# Threaded output mode?  0/1/N threads (from --threads)\n";
+        *of << "# Threaded output mode?  1/N threads (from --threads)\n";
         cmake_set_raw(*of, name + "_THREADS", cvtToStr(v3Global.opt.threads()));
```

Comment:
We should be able to delete this (Like in V3EmitMk.cpp)

Reply:
This is actually needed and it is different than ``VM_THREADS`` from ``V3EmitMk``.

It tells how many threads to pass into Verilator invoked via CMake and ``VM_THREADS`` where just telling to define ``VL_THREADED``. I've restored this.

Reply:
Hmm, curious why we actually need the thread count them?

Reply:
It is used when verilator emits CMake for e.g. hierarchical verilation, it passes value of original ``-threads`` to the generated CMake.

Anyway we shouldn't fail without this, so I found out that now, we need to always link MT libraries in CMake:
https://github.com/verilator/verilator/pull/3703/commits/134604f94adba592c91357de17077f1fd820ce25#diff-4ad84c02760f25fc00a01f346fe39a95903f6374320d4769243b557f99fbbb78L327